demo/handler.h

```diff
@@ -66,51 +66,53 @@ class Handler : public content_analysis::sdk::AgentEventHandler {
           event->GetResponse(),
           std::string(),
           content_analysis::sdk::ContentAnalysisResponse::Result::FAILURE);
-      std::cout << "  Verdict: failed to reach verdict: ";
-      std::cout << event->DebugString() << std::endl;
+      stream << "  Verdict: failed to reach verdict: ";
+      stream << event->DebugString() << std::endl;
     } else if (block) {
       auto rc = content_analysis::sdk::SetEventVerdictToBlock(event.get());
-      std::cout << "  Verdict: block";
+      stream << "  Verdict: block";
       if (rc != content_analysis::sdk::ResultCode::OK) {
-        std::cout << " error: "
-                  << content_analysis::sdk::ResultCodeToString(rc) << std::endl;
-        std::cout << "  " << event->DebugString() << std::endl;
+        stream << " error: "
+               << content_analysis::sdk::ResultCodeToString(rc) << std::endl;
+        stream << "  " << event->DebugString() << std::endl;
       }
-      std::cout << std::endl;
+      stream << std::endl;
     } else {
-      std::cout << "  Verdict: allow" << std::endl;
+      stream << "  Verdict: allow" << std::endl;
     }
 
-    std::cout << std::endl;
+    stream << std::endl;
 
     // If a delay is specified, wait that much.
     if (delay_ > 0) {
-      std::cout << "[Demo] delaying request processing for " << delay_ << "s" << std::endl;
+      //stream << "[Demo] delaying request processing for " << delay_ << "s" << std::endl;
```

Comment:
Is commenting out this line intentional?

Reply:
Good catch!  I meant to move this to `ProcessRequests()`.  Done.